rl_studio/config/config.yaml

```diff
@@ -181,7 +181,7 @@ environments:
   simple:
     env_name: F1Env-v0
     circuit_name: simple
-    training_type: dqn #qlearn_camera, ddpg, dqn
+    training_type: qlearn_camera #qlearn_camera, ddpg, dqn
```

Comment:
Not sure how we should handle this kind of modifications. It is an open discussion I guess.

As I understand, We decided to have just one file to configure all the parameters our project needs to run. That's right.

However, we didn't discuss yet if we should have different versions of this file for the different projects we are running (this is what I am doing) or if we should have just one file with all of the different projects inside and indicate what part of this file our execution must read according to the project we indicate at launching time.

If we go for the first one, I would create a new file called config_f1_qlearn.yaml of config_f1.yaml to keep the stable version of this project in the repo for a future joiner.
Otherwise,  we might need to remove all the other config files and include there here. 
What do you think it could be a better approach?
Do you think we should also provide different configuration files versions for the same project? (let's say, stable config file for dqn case, stable config file for qlearn case, etc.)

Thanks!

Reply:
I would think that the idea is to have a single config.yaml file where the parameters of each task are configured.
The problem that arises here is that there are many configuration parameters and when changing to other tasks or other algorithms when it is not done well, errors occur because RL-Studio is not designed to handle errors correctly.

However, as we are researching and therefore continually modifying the code, I personally find it quicker to create a config file for every single task. For example, config_f1_followlane_dqn.yaml indicates that it is a Follow Lane project with the F1 agent and the DQN algorithm. And for anyone else, it is easier to replicate the results.

I would think that the best approach is to use a single config, but we still don't have one designed and accepted by everyone. In addition, new tasks arise that make us rethink how to design it.

In all the config I also have, the structure is the same, just different values. That way, it's as easy as calling a preconfigured file. I would think, now, it is the best approach.

Reply:
Thanks for the comments!

I think that the best approach is to have separated config files at least for each task (we could even create folders for each task variant). Since the tasks can be quite different in nature, I don't know if the same config file can be used for each case.

I will update the config file I submitted with a new name and also add it to the installation instructions. I think it's important to also keep it easy for a newcomer to install everything and train an algo.

Do you have any idea for a naming convention that we should follow when naming the config files @pjfernandecabo @rubenlucas93? My suggestion is something like this:

```
config_TASK_ALGORIGHTM.yaml 
```

Reply:
I have updated the PR with the ideas I just suggested in the previous message

Reply:
Like it!
I approve :)

Reply:
merge when you want

Reply:
> Thanks for the comments!
> 
> I think that the best approach is to have separated config files at least for each task (we could even create folders for each task variant). Since the tasks can be quite different in nature, I don't know if the same config file can be used for each case.
> 
> I will update the config file I submitted with a new name and also add it to the installation instructions. I think it's important to also keep it easy for a newcomer to install everything and train an algo.
> 
> Do you have any idea for a naming convention that we should follow when naming the config files @pjfernandecabo @rubenlucas93? My suggestion is something like this:
> 
> ```
> config_TASK_ALGORIGHTM.yaml 
> ```

I would name it: config_AGENT_TASK_ALGORITHM.yaml